py/tests/tests_unit_conversions.py

```diff
@@ -189,5 +190,21 @@ def test_sparse_data_conversion(self):
 
         assert total < 10 # The conversion should not take longer then 10 seconds
 
+    def test_load_mojo_gbm(self):
+        from pysparkling.ml import H2OMOJOModel, H2OGBM
+        mojo = H2OMOJOModel.create_from_mojo("../ml/src/test/resources/binom_model_prostate.mojo")
```

Comment:
is this file in repo?

Reply:
yup, it's the same one we are using for tests in `H2OMojoTests`